build/packages.json

```diff
@@ -26,6 +16,16 @@
         "features/polyPinion",
         "features/polyPreview",
         "features/test",
-        "podjs"
+        "platform/feature-api/api/fetch-spec",
```

Comment:
I guess that the rationale for doing this is that these platform are not used directly by features, but by the platform, and that's probably mostly true. But I'd need to check that.

Reply:
Yup, pod.js is never used in production, that's why I removed it while at it